dds/DCPS/RTPS/RtpsDiscovery.cpp

```diff
@@ -392,24 +393,35 @@ RtpsDiscovery::Config::discovery_config(ACE_Configuration_Heap& cf)
           }
 #endif /* OPENDDS_SECURITY */
         } else if (name == "MaxSpdpTimerPeriod") {
-          // In milliseconds.
-          const OPENDDS_STRING& string_value = it->second;
-          int int_value;
-          if (DCPS::convertToInteger(string_value, int_value)) {
-            discovery->max_spdp_timer_period(ACE_Time_Value(0, int_value * 1000));
-          } else {
-            ACE_ERROR_RETURN((LM_ERROR,
-                              ACE_TEXT("(%P|%t) RtpsDiscovery::Config::discovery_config(): ")
-                              ACE_TEXT("Invalid entry (%C) for MaxSpdpTimerPeriod in ")
-                              ACE_TEXT("[rtps_discovery/%C] section.\n"),
-                              string_value.c_str(), rtps_name.c_str()), -1);
-          }
-        }  else {
+            // In milliseconds.
+            const OPENDDS_STRING& string_value = it->second;
+            int int_value;
+            if (DCPS::convertToInteger(string_value, int_value)) {
+              discovery->max_spdp_timer_period(ACE_Time_Value(0, int_value * 1000));
+            } else {
+              ACE_ERROR_RETURN((LM_ERROR,
+                                ACE_TEXT("(%P|%t) RtpsDiscovery::Config::discovery_config(): ")
+                                ACE_TEXT("Invalid entry (%C) for MaxSpdpTimerPeriod in ")
+                                ACE_TEXT("[rtps_discovery/%C] section.\n"),
+                                string_value.c_str(), rtps_name.c_str()), -1);
+            }
+        } else if (name == "MaxSpdpSequenceMsgResetChecks") {
+            const OPENDDS_STRING& string_value = it->second;
+            u_short value;
```

Comment:
@mitza-oci This definitely not something that needs to be changed right now, since we're already using `u_short` and `u_long` but this is the first time I've really noticed them. I tried googling them and they aren't standard C++ primitive names and we don't seem to be handling them in ACE so I don't feel comfortable with them from a portability perspective. However they probably won't ever be a problem though since it seem so many platforms seem to define them, but I just want to complain I guess...

Reply:
They are defined in ACE (ace/os_include/os_stdint.h), so no problem using them here.  Wouldn't necessarily want to see them in public interfaces.

Reply:
Indeed they are. Looks like I miss typed `typedef` when greping my grep for `u_long` in ACE. I'm officially withdrawing my complaint.